src/components/control/control.svelte

```diff
@@ -0,0 +1,106 @@
+<script lang="ts" context="module">
+  export let sizes = ['small', 'normal'] as const
+  export type Size = (typeof sizes)[number]
+</script>
+
+<script lang="ts">
+  export let disabled = false
+  export let size: Size = 'normal'
+  export let required = false
```

Comment:
I like the idea of this - common functionality for form-style components. But perhaps "Control" is too generic and similar to "Component". I wouldn't classify a Checkbox as not a control, for example. How about "FormItem" or "LabeledComponent" or similar?

Reply:
Renamed to 'FormItem'